test/specs/helpers.canvas.tests.js

```diff
@@ -21,6 +21,16 @@ describe('Chart.helpers.canvas', function() {
       expect(chart.ctx.clearRect.calls.first().object).toBe(chart.ctx);
       expect(chart.ctx.clearRect.calls.first().args).toEqual([0, 0, 150, 245]);
     });
+
+    it('should not throw error when chart is null', function() {
+      function createChart() {
+        return acquireChart({}, {
+          canvas: null
+        });
+      }
+
+      expect(createChart).not.toThrow();
```

Comment:
Shouldn't we check on `helpers.clearCanvas` to not throw since that is the method that broke instead of the whole create canvas.

If for some reason the method does not get called in acquireChart this test will lose its value.

Reply:
Good catch @LeeLenaleee , added a call to `helpers.clearCanvas` in e1d670dbbff8efa41772677cdcb9e448d78f73c0